### PR TITLE
Giveme

### PR DESCRIPTION
**Pull request by robinkanters**
/giveme <x> [y]

can now be used as alias for

    /give <your username> <x> [y]

### CODE REVIEW

File: TrueCraft/Commands/GiveCommand.cs [hunk 1]
```diff
@@ -39,47 +38,59 @@ public override void Handle(IRemoteClient client, string alias, string[] argumen
             if(arguments.Length >= 4)
                     amount = arguments[3];
             
+            var receivingPlayer = GetPlayerByName(client, username);
+
+            if (receivingPlayer == null)
+            {
+                client.SendMessage("No client with the username \"" + username + "\" was found.");
```
Comment (SirCmpwn) [inline TrueCraft/Commands/GiveCommand.cs]: "client" isn't a very user-friendly term. How about "That player is not logged in"

File: TrueCraft/Commands/GiveCommand.cs [hunk 2]
```diff
@@ -39,47 +38,59 @@ public override void Handle(IRemoteClient client, string alias, string[] argumen
...
+                Help(client, alias, arguments);
+            }
+        }
+
+        protected static IRemoteClient GetPlayerByName(IRemoteClient client, string username)
+        {
             var receivingPlayer =
-                client.Server.Clients.FirstOrDefault(c => String.Equals(c.Username, username, StringComparison.CurrentCultureIgnoreCase));
+                client.Server.Clients.FirstOrDefault(
+                    c => String.Equals(c.Username, username, StringComparison.CurrentCultureIgnoreCase));
```
Comment (SirCmpwn) [inline TrueCraft/Commands/GiveCommand.cs]: I prefer `c => c.ToUpper() == username.ToUpper()`

**Comment by SirCmpwn** [comment]
Looks good save for those minor issues.

**Comment by robinkanters** [comment]
those 'issues' both didn't come from this PR

**Comment by SirCmpwn** [comment]
True, but it's a good time to fix them, don't you think?

**Comment by robinkanters** [comment]
Perhaps, but it's outside of the scope of this PR

**Comment by SirCmpwn** [comment]
Whatever. LGTM.

**Pull request merged**